test/conformance/kuttl/verifyImages/clusterpolicy/standard/rollback-image-verification/policy.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: verify-images
+---
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-image
+spec:
+  validationFailureAction: Enforce
+  background: false
+  webhookTimeoutSeconds: 30
+  failurePolicy: Fail
+  rules:
+    - name: check-image
+      match:
+        any:
+        - resources:
+            kinds:
+              - Deployment
+      verifyImages:
+      - imageReferences:
+        - "ghcr.io/hackeramitkumar*"
```

Comment:
Can we change the image from the Kyverno registry? I can help build and push the images.